src/main/webapp/app/shared/user-settings/user-settings-container/user-settings-container.component.ts

```diff
@@ -33,7 +37,16 @@ export class UserSettingsContainerComponent implements OnInit {
 
         this.accountService
             .getAuthenticationState()
-            .pipe(tap((user: User) => (this.currentUser = user)))
+            .pipe(
+                tap((user: User) => {
+                    this.currentUser = user;
+                    this.authorizeTutor();
+                }),
+            )
             .subscribe();
     }
+
+    authorizeTutor() {
```

Comment:
I think this should be part of the `account.service.ts` since this might be reusable. Then using `hasAnyAuthorityDirect(['ROLE_TUTOR', 'ROLE_EDITOR', …])` internally instead would also make more clear what is checked.

Reply:
That's a nice idea, I'll change that :+1: